README.md

```diff
@@ -75,15 +75,15 @@ precedence over options specified in one of these config files. Here are some
 sample configs that you can copy into your project to get started:
 
 _pyproject.toml_
-```
+```toml
 [tool.pytest.ini_options]
 retries = 2
 retry_delay = 0.5
 cumulative_timing = false
 ```
 
 _config.ini/tox.ini_
-```
+```toml
```

Comment:
Is there association for .ini? That would be the correct value here if so. Otherwise, I suppose toml works.

Reply:
```suggestion
```ini
```

Reply:
```suggestion
```ini
```